components/lib/fileupload/FileUpload.css

```diff
@@ -23,6 +23,13 @@
     left: 0;
 }
 
+.p-fileupload-content.p-fileupload-highlight {
+    border-color: var(--primary-color);
+    border-width: 2px;
+    border-style: dashed;
+    background-color: var(--primary-100);
+}
```

Comment:
I think this change belongs in the PrimeReact SASS Theme project: https://github.com/primefaces/primereact-sass-theme

Reply:
So I should create a PR there adding that line and removing this @melloware?
I didn't understand what lines I keep here, and what lines I add in a new PR in PrimeReact SASS Theme project.
I don't know if is adding in the sass projects because always when I fixed things in PrimeNG project, nobody told me that I have to do it in SASS Theme and were approve. Now I have my doubts 😞

Reply:
Good question.   If they approved it on PrimeNG side maybe it is OK here?

Reply:
But usually any styling involving `vars` goes in the SASS theme project and not in the core CSS of the lib.

Reply:
> But usually any styling involving `vars` goes in the SASS theme project and not in the core CSS of the lib.

Is not approve yet in PrimeNG but others PR that I did fixing styles nobody told me anything about the Sass Theme Project and from what I understand (maybe I'm wrong), the sass project was released and now it's free but I don't know if it's still being maintained or not because I read that there will be a paid version in the future with new changes

Reply:
Well do me a favor open a PR in the PrimeReact Sass theme with just the 1 change there so we can point it back to this ticket as well.

Reply:
> Well do me a favor open a PR in the PrimeReact Sass theme with just the 1 change there so we can point it back to this ticket as well.

Done, you have the PR here @melloware. Thanks!